### DISCUSSION TITLE
Callback query

### DISCUSSION THREAD

**Discussion by Arc9067** [eternnoir/pyTelegramBotAPI#1438] [Q&A] [answered]
```
chart_main.add(types.InlineKeyboardButton(text='🔁 Refresh', callback_data= 'refresh'))

```

have set up this, but confused about how to refresh the text

**Comment by coder2020official** [comment]
Create a callback query handler, check data, and edit the buttons..

**Reply by Arc9067** [reply]
How to edit the commands, that's what is confusing me

**Comment by Badiboy** [comment, marked as answer]
You cannot edit buttons. You can only edit message and set new inline keyboard with
editMessageReplyMarkup
